easybuild/easyblocks/m/mvapich2.py

```diff
@@ -70,12 +70,24 @@ def configure_step(self):
 
         # enable specific support options (if desired)
         if self.cfg['withmpe']:
-            add_configopts.append('--enable-mpe')
+            # --enable-mpe is coming from MPICH.
```

Comment:
I find this confusing, since the MPICH easyblock is unaware of `--enable-mpe`...

maybe say this is a configure option for all MPICH-based MPI libraries?